core/ui/PageTemplate/sidebar.tid

```diff
@@ -6,18 +6,22 @@ tags: $:/tags/PageTemplate
 $:/config/SideBarSegments/Visibility/$(listItem)$
 \end
 
-<$eventcatcher tag="div" class="tc-sidebar-resizer-pointerdown-eventcatcher" selector=".tc-sidebar-resizer" matchSelector=".tc-sidebar-resizer" $pointerdown=<<sidebar-resizer-pointerdown-actions>> $pointerup=<<sidebar-resizer-pointerup-actions>>>
+<$eventcatcher tag="div" class="tc-sidebar-resizer-pointerdown-eventcatcher" selector=".tc-sidebar-resizer" matchSelector=".tc-sidebar-resizer" $pointerdown=<<sidebar-resizer-pointerdown-actions>>>
 
 <%if [{$:/state/sidebar}!match[no]] %>
```

Comment:
is `!match[no]` === `match[yes]` -- I would prefer a logic, that it easier to understand

Reply:
This check is perfectly fine @pmario and it needs to be like this.
`$:/state/sidebar` can be missing like it does at startup and generally the sidebar is only hidden if the text of `$:/state/sidebar` is "no"